tests/unit/gpu_av_buffer_device_address_positive.cpp

```diff
@@ -2396,3 +2396,82 @@ TEST_F(PositiveGpuAVBufferDeviceAddress, GlobalInvocationIdIVec3) {
     pipe.cs_ = std::make_unique<VkShaderObj>(this, shader_source, VK_SHADER_STAGE_COMPUTE_BIT, SPV_ENV_VULKAN_1_0, SPV_SOURCE_ASM);
     pipe.CreateComputePipeline();
 }
+
+TEST_F(PositiveGpuAVBufferDeviceAddress, DualShaderLibrary) {
+    TEST_DESCRIPTION("Mimics issue found with real world GPL usage");
```

Comment:
```suggestion
    TEST_DESCRIPTION("Mimics issue found with real world GPL usage");
```
Maybe add a brief description of the issue for those new tests?